src/library-authoring/hierarchy/ContainerHierarchy.tsx

```diff
@@ -0,0 +1,41 @@
+import Loading from '@src/generic/Loading';
+import { useSidebarContext } from '../common/context/SidebarContext';
+import { useContainerHierarchy } from '../data/apiHooks';
+import { ItemHierarchy } from './ItemHierarchy';
+
+export const ContainerHierarchy = ({
```

Comment:
Do you think it is worth merging `ComponentHierarchy` and `ContainerHierarchy`?

To avoid handling two hooks, you could also use only one hook (such as `useItemHierarchy`), which handles the correct API call for items and containers.

Reply:
Yes, thanks for the recommendation. Updated: https://github.com/openedx/frontend-app-authoring/pull/2399/commits/8c45e4d462aa1dd3c03f6e842416758742cd6eee